### DISCUSSION TITLE
cannot override firefox shortcuts

### DISCUSSION THREAD

**Discussion by txtyash** [gdh1995/vimium-c#534] [Q&A] [answered]
I'm unable to override firefox shortcuts. Is there a way to achieve it?

**Comment by gdh1995** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/5547703/148505493-9264fe63-d589-4929-b3e5-1c49689d535d.png)

![image](https://user-images.githubusercontent.com/5547703/148505514-99f0b199-8e8a-49d0-9eaa-ef2322e14a18.png)

Wiki: https://github.com/gdh1995/vimium-c/wiki/Trigger-commands-in-an-input-box#user-content-shortcut
